source/2020-09-25-the-ember-times-issue-164.md

```diff
@@ -0,0 +1,204 @@
+---
+title: The Ember Times - Issue No. 164
+author: Abhilash LR, Chris Ng, Jared Galanis, Isaac Lee, Amy Lam
+tags: Recent Posts, Newsletter, Ember.js Times, Ember Times, 2020
+alias : "blog/2020/09/25-the-ember-times-issue-164.html"
+responsive: true
+---
+
+👋 Emberistas! 🐹
+
+Hacktoberfest + The Ember Times ✍️,
+A new reference page for Ember CLI commands 🔖,
+migrating off of mixins in Ember Octane 👋,
+async data and autotracking in Ember Octane ✨,
+autotracking developer experience 🔥,
+ember-concurrency 2.0.0.beta-1 🎉,
+and last, but not least, EmberJS with REST API 💪!
+
+READMORE
+
+---
+
+## [Hacktoberfest + The Ember Times ✍️](https://github.com/ember-learn/ember-blog/blob/master/CONTRIBUTING.md#writing-for-the-ember-times)
+
+<!--alex ignore host-hostess-->
+[Hacktoberfest](https://hacktoberfest.digitalocean.com/) is a celebration of open source open to everyone in the global GitHub community. Whether you’re a developer, student learning to code, event host, or company of any size, you can help drive growth of open source and make positive contributions to an ever-growing community. All backgrounds and skill levels are encouraged to complete the challenge!
+
+Need ideas for Hacktoberfest contributions? Or looking to get started in open source? We suggest working with us on the [next Ember Times issue](https://github.com/ember-learn/ember-blog/pull/778)! As you may know, The Ember Times is the newsletter for the official [Ember Blog](https://blog.emberjs.com/). As writer, you will take on an active role in the Ember community and keep up with the latest events and discoveries. Prior experience in technical writing isn't required, as we will be happy to mentor and pair with you. We also encourage everyone around the globe to join; don't worry if English is your second language (or third, etc.).
```

Comment:
Perfect transition!

Reply:
@ijlee2 thanks for all of your help as always, much appreciated!! 💯

Reply:
teamwork makes the dreamwork!